server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -671,6 +671,11 @@ static boolean isDir(String delete) {
     return slashCount == 1;
   }
 
+  @Override
+  public String getHostname() {
```

Comment:
I believe this is supposed to return the service address stored in ZooKeeper, not the bind address. It might make sense to move this back to the abstract super class.

Reply:
See previous comment but if the hostname assignment is moved back to the Abstract class then what is going on may be more clear.